src/tests/ftest/datamover/dm_dst_create.yaml

```diff
@@ -26,15 +26,9 @@ ior:
     transfer_size: '1K'
     signature: 5
 dcp:
-    type_mux: !mux
-        type_posix_dfs:
-            cont_type: POSIX
-            daos_api: DFS
-        type_posix_obj:
-            cont_type: POSIX
-            daos_api: DAOS
-        type_unknown_obj:
-            daos_api: DAOS
+    client_processes:
+        np: 3
+fs_copy:
```

Comment:
fs_copy doesn't need client_processes because it only uses a single process.
This just corresponds to mpi ranks.